### PR TITLE
Move hardcoded URLS to environment.ts

### PR DESCRIPTION
**Pull request by bisbaldi**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
-
+import {NgModule} from '@angular/core';
```
Comment (fyndalf) [inline src/app/app.module.ts]: Fix spacing between import argument to be `import { someModule } from ...`

File: src/app/services/private-stats.service.ts [hunk 2]
```diff
@@ -1,16 +1,14 @@
 import {Inject, Injectable} from '@angular/core';
 import {HttpClient} from '@angular/common/http';
+import {environment} from '../../environments/environment';
```
Comment (fyndalf) [inline src/app/services/private-stats.service.ts]: Fix spacing

File: src/app/services/public-stats.service.ts [hunk 3]
```diff
@@ -1,16 +1,14 @@
 import {Inject, Injectable} from '@angular/core';
 import {HttpClient} from '@angular/common/http';
+import {environment} from '../../environments/environment';
```
Comment (fyndalf) [inline src/app/services/public-stats.service.ts]: Also fix spacing here

File: src/styles.css [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
+  background-size: cover;
+  background-repeat: no-repeat;
+}
 
 #form {
-    color:black !important;
-  }
\ No newline at end of file
+  color: black !important;
+}
```
Comment (fyndalf) [inline src/styles.css]: Insert trailing newline

**Review by fyndalf — CHANGES_REQUESTED** [review]
I've added some comments - after the changes have been made, this will look good to me :)

**Review by svitlanakalancha — APPROVED** [review]

**Review by fyndalf — APPROVED** [review]

**Pull request merged**